app/controllers/applications_controller.rb

```diff
@@ -4,10 +4,21 @@ def new
   end
 
   def create 
-    flash[:notice] = 'Your Application has been submitted successfully!'
-    favorites.update_favorites(params[:pet_ids])
-    redirect_to "/favorites"
+    application = Application.new(favorites_params)
+    if (application.save) && (!params[:pet_ids].nil?)
```

Comment:
Used to make sure not only are all the text fields filled out but that at least one favorited pet is checked off as well for the application.